index.js

```diff
@@ -74,10 +74,10 @@ module.exports = bundler => {
 
         // recursive copy function
         let numWatches = 0;
-        const copyDir = (staticDir, bundleDir) => {
+        const copyDir = (staticDir, bundleDir, excludeGlob) => {
             if (fs.existsSync(staticDir)) {
                 const copy = (filepath, relative, filename) => {
-                    if (config.excludeGlob && config.excludeGlob.find(excludeGlob =>
+                    if (excludeGlob.find(excludeGlob =>
```

Comment:
If this is confusing with both variables named excludeGlob, lmk and I'll rename the second one `glob`

Reply:
It looks ok :)